packages/custom-field/test/visual/lumo/custom-field.test.js

```diff
@@ -134,26 +131,18 @@ describe('custom-field', () => {
         wrapper = fixtureSync(`
           <div>
             <vaadin-custom-field label="Custom field">
-              <vaadin-select value="+358">
-                <template>
-                  <vaadin-list-box>
-                    <vaadin-item>+358</vaadin-item>
-                    <vaadin-item>+46</vaadin-item>
-                    <vaadin-item>+34</vaadin-item>
-                  </vaadin-list-box>
-                </template>
-              </vaadin-select>
+              <vaadin-select
+                value="+358"
+                items='[{ "label": "+358" }, { "label": "+46" }, { "label": "+34" }]'
```

Comment:
Using single quotes here is needed for Polymer to correctly decode attribute value as JSON array.